compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  web:
+    container_name: govuk-frontend-flask
+    build:
+      context: .
+      args:
+        BASE_IMAGE: python:3.12-bookworm
+      target: development
+    restart: always
+    environment:
+      - SECRET_KEY=CHANGE_ME
+    ports:
+      - "8000:8000"
```

Comment:
Should this default to a port other than 8000, this conflicts with the default `cla_public` port.